lib/kafkaesque.js

```diff
@@ -18,10 +18,15 @@ var assert = require('assert');
 var api = require('./api');
 var events = require('events');
 var _ = require('underscore');
+var reInterval = require('reInterval');
 
-var MAX_WAIT_DEFAULT = 5000;
-var MIN_BYTES_DEFAULT = 2;
 
+// _options = {
```

Comment:
Can this whole file be split up better? It's nearly 1000 lines long, very hard to review code like this which means things will slip past. Are there any easy places to break this out?